diktat-api/src/main/kotlin/com/saveourtool/diktat/DiktatRunnerFactory.kt

```diff
@@ -5,14 +5,20 @@ import com.saveourtool.diktat.api.DiktatBaselineFactory
 import com.saveourtool.diktat.api.DiktatProcessorListener
 import com.saveourtool.diktat.api.DiktatReporter
 import com.saveourtool.diktat.api.DiktatReporterFactory
+import com.saveourtool.diktat.api.DiktatRuleConfig
 import com.saveourtool.diktat.api.DiktatRuleConfigReader
+import com.saveourtool.diktat.api.DiktatRuleSet
 import com.saveourtool.diktat.api.DiktatRuleSetFactory
 import java.io.OutputStream
 import java.nio.file.Path
 
 /**
  * A factory to create [DiktatRunner]
  *
+ * @property diktatRuleConfigReader a reader for [DiktatRuleConfig]
```

Comment:
actually, read https://stackoverflow.com/questions/45032436/what-is-the-difference-between-properties-and-parameters-in-kotlin one and looks line the private property is `@param`

@akuleshov7 what do you think?

Reply:
discussed offline with @akuleshov7  -- it should be `@param`

Reply:
Done